stdlib/fractions.pyi

```diff
@@ -20,11 +20,23 @@ else:
     @overload
     def gcd(a: Integral, b: Integral) -> Integral: ...
 
+_IR = TypeVar("_IR", covariant=True)
+
+class _ConvertibleToIntegerRatio(Protocol[_IR]):
+    def as_integer_ratio(self) -> _IR: ...
```

Comment:
nit: I'd just call this TypeVar `_T_co`. `_IR` makes me think that it's going to be bound to a protocol -- but instead, the protocol is generic over the TypeVar

```suggestion
_T_co = TypeVar("_T_co", covariant=True)

class _ConvertibleToIntegerRatio(Protocol[_T_co]):
    def as_integer_ratio(self) -> _T_co: ...
```